util/log/clog.go

```diff
@@ -37,6 +37,7 @@ import (
 	"time"
 
 	"github.com/cockroachdb/cockroach/proto"
+	"github.com/cockroachdb/cockroach/util"
```

Comment:
you're right. Fine leaving it as is for now.
Shouldn't it be easy (and a good idea) to take the logging-related protos out of `proto` and put them in `util/log/logpb` or something like that, though?